pelican/readers.py

```diff
@@ -347,6 +351,12 @@ def read(self, source_path):
             metadata = {}
         return content, metadata
 
+    def disabled_message(self) -> str:
+        return (
+            "Could not import markdown.Markdown.  "
+            "Have you installed the markdown package?"
```

Comment:
Total nitpick, but since the package name is actually upper-cased — and because capitalized proper nouns make sentences more readable — I suggest: `markdown package` ➡️ `Markdown package`

Reply:
I'm actually confused on this point.

https://github.com/Python-Markdown/markdown says `pip install markdown`, i.e. lowercase. To me, the package name is the thing after "pip install". The string in pyproject.toml is also lowercase.

However, pip freeze shows uppercase:

```
$ pip freeze | grep -i mark
Markdown==3.5.1
markdown-it-py==3.0.0
MarkupSafe==2.1.3
```

I'll do whatever you want. Just let me know.

Reply:
I'm with boxydog on this one; I prefer the string to match the one you pip install.

Reply:
Not a big deal either way — we can leave it as-is.

Some info re: my reasoning… I personally prefer uppercase proper nouns in almost all cases, as I find lowercase proper nouns to be harder to visually parse in the context of a sentence. In a richer-text context, that can be ameliorated by using `code formatting` or [links](https://github.com/getpelican/pelican/pull/3321), but that doesn't work in plain-text contexts such as this one.